topics/master/yaml_processing_extensions.txt

```diff
@@ -0,0 +1,77 @@
+YAML Processing Extensions
+---------------------------------------------------------------------------------------------------
+
+Variable back-referencing:
+------------------------------
+
+In order to reduce repetition in .yml property files Hodor has implemented variable back referencing.
```

Comment:
I have a few sentences I'd like to contribute to this explanation. Will add that here shortly.